ui/src/app/components/results/test-case-result-details.component.html

```diff
@@ -69,20 +69,42 @@
             [class.mouse-over]="testCaseResult?.testCase?.preRequisite">
             <button
               [disabled]="startSync"
-              *ngIf="(testCaseResult.iteration && !testCaseResult.parentResult?.isExecuting||!testCaseResult.iteration) && !testCaseResult?.isExecuting && isDry"
+              *ngIf="(testCaseResult.iteration && !testCaseResult.parentResult?.isExecuting||!testCaseResult.iteration) && !testCaseResult?.isExecuting && isDry && activeTab=='steps'"
```

Comment:
plz avoid using hard coded string "steps", it should be configurable.

Reply:
Done